sky/build/sky_app.gni

```diff
@@ -129,7 +131,41 @@ template("sky_app") {
         deps += invoker.deps
       }
     }
-  } else if(is_ios) {
+  } else if(is_mac && defined(invoker.info_plist)) {
+    import("//build/config/mac/rules.gni")
+
+    mac_app(target_name) {
+
+      app_name = target_name + "_app"
+      info_plist = invoker.info_plist
+      xibs = invoker.xibs
+
+      resources_gen_target_name = target_name + "_res"
+      resource_copy_mac(resources_gen_target_name) {
+        resources = [
+          "//third_party/icu/android/icudtl.dat",
```

Comment:
Ideally we'd share this dependency with sky_shell isntead of having to duplicated it to sky_app.